integrations/standalone/tests/mock/dataclass-editor.spec.ts

```diff
@@ -1,14 +1,28 @@
-import { test } from '@playwright/test';
+import { expect, test } from '@playwright/test';
 import { DataClassEditor } from '../pageobjects/DataClassEditor';
 
-test.describe('DataClassEditor', () => {
-  let editor: DataClassEditor;
+let editor: DataClassEditor;
 
-  test.beforeEach(async ({ page }) => {
-    editor = await DataClassEditor.openMock(page);
-  });
+test.beforeEach(async ({ page }) => {
+  editor = await DataClassEditor.openMock(page);
+});
```

Comment:
Is it fine for playwright tests to do this?

Reply:
I think so, because it doesn't hold any state, it only provides some helper functions for testing.